lib/otp/views/configure_yubiotp_dialog.dart

```diff
@@ -34,6 +34,7 @@ import '../../widgets/app_text_field.dart';
 import '../../widgets/choice_filter_chip.dart';
 import '../../widgets/responsive_dialog.dart';
 import '../keys.dart' as keys;
+import '../keys.dart';
```

Comment:
remove this and use the prefix (instead of useSerial use keys.useSerial and so on)